client/README.md

```diff
@@ -224,6 +172,89 @@ following shell scripts for convenience:
 - `clean.sh`: Removes local `preprocessed`, `translated` and `postprocessed`
   directories.
 
+
+## Running Using `bqms-run` Directly
+
+Direct execution is suitable for advanced use cases. It requires setting
+[environment variables](#environment-variables) and modifying a
+[config.yaml](#configyaml) file, then running the `bqms-run` command
+installed by this repository's `pip` package.
+
+A Docker container of `bqms-run` with all the necessary dependencies can be
+built by running:
+
+```shell
+sudo docker build -t bqms-run client
+```
+
+See also the instructions for [deploying to Cloud Run](#deploying-to-cloud-run).
+
+### Environment Variables
```

Comment:
The Env Vars and the config.yaml sections are also important to first time users. I think moving this section upper on this page might be easier for beginner to get started.